data/channels.csv

```diff
@@ -2918,106 +2918,106 @@ BelqeesTV.ye,Belqees TV,قناة بلقيس,,Tawakkol Karman,YE,,Istanbul,c/YE,a
-BeMad.es,Be Mad,,,,ES,,,c/ES,spa,,FALSE,2016-01-01,,,https://www.bemad.es/,https://upload.wikimedia.org/wikipedia/commons/thumb/7/70/Be_Mad_TV.svg/512px-Be_Mad_TV.svg.png
+BeMad.es,Be Mad,,,Mediaset Espana,ES,,Madrid,c/ES,spa,movies,FALSE,2016-04-21,,,https://www.bemad.es/,https://upload.wikimedia.org/wikipedia/commons/thumb/7/70/Be_Mad_TV.svg/512px-Be_Mad_TV.svg.png
 BemTV.br,Bem TV,,SBT,,BR,BR-MT,Tangará da Serra,s/BR-MT,por,general,FALSE,,,,https://www.bemtv.online/,https://i.imgur.com/ifGQGJx.png
-BenficaTV.pt,Benfica TV,,,S.L. Benfica,PT,,,c/PT,por,,FALSE,2008-10-02,,,https://btvonline.nos.pt/,https://i.imgur.com/l6LACto.png
+BenficaTV.pt,Benfica TV,,,S.L. Benfica,PT,,Lisbon,c/PT,por,sports,FALSE,2008-10-02,,,https://btvonline.nos.pt/,https://i.imgur.com/l6LACto.png
 BenghaTV.cm,Bengha TV,,,,CM,,,c/CM,fra,,FALSE,,,,,https://i.imgur.com/oBAFwyv.png
-BenguturkTV.tr,Benguturk TV,Bengütürk TV,,,TR,,,c/TR,tur,,FALSE,,,,https://www.benguturk.com/,https://i.imgur.com/p3ON1oX.png
-BenieTV.ci,Benie TV,,,,CI,,,c/CI,fra,,FALSE,,,,https://www.benie.tv/,https://i.imgur.com/4dGwv7Q.png
+BenguturkTV.tr,Benguturk TV,Bengütürk TV,,,TR,,Istanbul,c/TR,tur,news,FALSE,,,,https://www.benguturk.com/,https://i.imgur.com/p3ON1oX.png
+BenieTV.ci,Benie TV,,,,CI,,Abidjan,c/CI,fra,religious,FALSE,,,,https://www.benie.tv/,https://i.imgur.com/4dGwv7Q.png
 BeninWebTV.bj,Benin Web TV,Bénin Web TV,,,BJ,,Cotonou,c/BJ,fra,news,FALSE,,,,https://beninwebtv.com/,https://i.imgur.com/CYy4n0N.png
 BensenvilleTV.us,BensenvilleTV,,,,US,US-IL,Bensenville,s/US-IL,eng,general,FALSE,,,,https://www.bensenville.il.us/713/BensenvilleTV-Live-Web-Stream,https://i.imgur.com/X7p0s2J.png
 BENTelevision.uk,BEN Television,Bright Entertainment Network,,,UK,,London,c/UK,eng,entertainment,FALSE,,,,https://bentelevision.com/,https://i.imgur.com/DUAjjvT.png
-BenTreTV.vn,Ben Tre TV,Bến Tre TV,,Ben Tre Radio and Television Station,VN,VN-50,,c/VN,vie,,FALSE,1985-07-22,,,http://thbt.vn/,https://i.postimg.cc/KYT0Czv3/Logo-THBT-B-n-Tre-m-i-2022-Logo-m-n-h-nh-B-n-1.png
+BenTreTV.vn,Ben Tre TV,Bến Tre TV,,Ben Tre Radio and Television Station,VN,VN-50,Ben Tre,c/VN,vie,general,FALSE,1985-07-22,,,http://thbt.vn/,https://i.postimg.cc/KYT0Czv3/Logo-THBT-B-n-Tre-m-i-2022-Logo-m-n-h-nh-B-n-1.png
 BenueTelevision.ng,Benue Television,,,,NG,,Makurdi,c/NG,eng,general,FALSE,,,,https://benuetv.be.gov.ng/,https://i.imgur.com/gP5aYgG.jpg
-BeratTV.tr,Berat TV,,,,TR,,,c/TR,tur,,FALSE,,,,,https://i.imgur.com/BxxH8wE.png
-BerbereTV.fr,Berbere TV,Berbère TV,,BRTV Group,FR,,Paris,c/FR;r/MAGHREB,ara;fra,culture,FALSE,,,,http://www.brtvpro.tv/,https://i.imgur.com/Uc2PAmL.png
-BerenteTV.hu,Berente TV,,,,HU,,,c/HU,hun,general,FALSE,,,,https://berente.hu,https://i.imgur.com/3H3Ct9f.png
+BeratTV.tr,Berat TV,,,,TR,,Istanbul,c/TR,tur,general,FALSE,,,,https://www.berattv.com.tr/,https://i.imgur.com/BxxH8wE.png
+BerbereTV.fr,Berbere TV,Berbère TV,,,FR,,Paris,c/FR,fra,culture,FALSE,,,,http://www.brtvpro.tv/,https://i.imgur.com/actf7dq.png

```

Comment:
Why replace the image with a lower quality?

Reply:
I don't remember doing that, Alstruit. Feel free to revert.